ui/package.json

```diff
@@ -7,18 +7,20 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "test:integration": "PORT=3000 wdio run ./wdio.conf.js",
+    "test:integration": "cypress run",
+    "cypress:open": "cypress open",
```

Comment:
ooo what does this do? Interactive debugging?

Reply:
yes that! try it out :)